### PR TITLE
add abbreviations for `‖ ‖` which place the cursor inside

### PR DESCRIPTION
**Pull request by j-loreaux**
This adds the abbreviations `norm` and `nnnorm` for `‖$CURSOR‖` and `‖$CURSOR‖₊`, respectively, for convenience.

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
LGTM, but we should wait to see whether #17575 gets merged and adjust the symbols accordingly.

File: src/abbreviation/abbreviations.json [hunk 1]
```diff
@@ -12,6 +12,8 @@
     "f<>": "‹$CURSOR›",
     "f<<>>": "«$CURSOR»",
     "[--]": "⁅$CURSOR⁆",
+    "nnnorm": "∥$CURSOR∥₊",
+    "norm": "∥$CURSOR∥",
```
Comment (eric-wieser) [inline src/abbreviation/abbreviations.json:16]: After https://github.com/leanprover-community/mathlib/pull/17575,
```suggestion
    "nnnorm": "‖$CURSOR‖₊",
    "norm": "‖$CURSOR‖",
```

**Comment by eric-wieser** [comment]
Mind editing the PR description and title to use the new characters?

**Comment by j-loreaux** [comment]
done, Eric.

**Comment by eric-wieser** [comment]
Thanks! Note I don't have permission to merge or even enable CI

**Pull request merged**